examples/websocket/market_monitoring.py

```diff
@@ -110,38 +110,17 @@ async def main():
 
     logger.info("Connecting to WebSocket asynchronously")
     logger.info("Press Ctrl+C to exit")
-    
-    try:
-        # Connect asynchronously - this will return immediately
-        await ws.async_connect()
-        
-        # Start our concurrent task
-        task = asyncio.create_task(periodic_task())
-        
-        # Main application loop
-        while True:
-            await asyncio.sleep(1)
-            
-    except KeyboardInterrupt:
-        logger.info("Exiting gracefully")
-    except Exception as e:
-        logger.error(f"Error: {e}")
-    finally:
-        # Cancel our concurrent task if it's still running
-        if 'task' in locals() and not task.done():
-            task.cancel()
-            try:
-                await task
-            except asyncio.CancelledError:
-                logger.info("Concurrent task was cancelled")
-                
-        logger.info("WebSocket connection closed")
```

Comment:
Are you sure, when you CTRL + C now, to stop the process, the terminal doesn't get flooded with the stack trace or anything like that? That's the reason why we were catching the KeyboardInterupt & Exception here